MmSupervisorPkg/SupervisorPolicyTools/policy_entry.py

```diff
@@ -166,14 +167,14 @@ def Decode(self, Buffer) -> bytes:
         # slice off buffer bytes decoded
         return Buffer[self._StructSize:]
 
-    def DumpInfo(self, prefix: str = "") -> None:
+    def DumpInfo(self, prefix: str = "", outfs: IO = sys.stdout) -> None:
         '''Print the object in human readable form.
 
         prefix: str printed before each line of output.  Best used for whitespace and alignment
         '''
-        print(f"{prefix}Version: 0x{self.Version:08X}")
-        print(f"{prefix}Type: {self.Type}")
-        print(f"{prefix}EntrySize: {self.EntrySize}")
+        outfs.write(f"{prefix}Version: 0x{self.Version:08X}\n")
+        outfs.write(f"{prefix}Type: {self.Type}\n")

```

Comment:
As mentioned, l think a logging.verbose should do this one, too.

Reply:
verbose is not a logging level as we only use the python provided logging levels. logging.DEBUG is the most detailed of the levels. Are there any changes you would like?